### DISCUSSION TITLE
Change bit Depth

### DISCUSSION THREAD

**Discussion by susanpettigrew** [ImageMagick/ImageMagick#2593] [Help]
Hi- I am trying to see if I can use ImageMagik to run a batch action to change the bit depth of files from 16bit to 8bit? Sorry- all the links I tried from the searches came back with file not found.
Susan

**Comment by snibgo** [comment]
This will do that:
```
magick mogrify -depth 8 *.png
```

**Reply by susanpettigrew** [reply]
Thank you

**Reply by fmw42** [reply]
See https://imagemagick.org/index.php
